sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/ScalaJSOutputAnalyzer.scala

```diff
@@ -54,4 +54,39 @@ object ScalaJSOutputAnalyzer {
       .distinct
   }
 
+  /**
+    * @return The list of ES modules exported by a Scala.js project
+    * @param linkerConfig Configuration of the Scala.js linker
+    * @param linker Scala.js linker
+    * @param irFiles Scala.js IR files
+    * @param moduleInitializers Scala.js module initializers
+    * @param logger Logger
+    */
+  def findTopLevelExports(
+                           linkerConfig: StandardLinker.Config,
+                           linker: ClearableLinker,
+                           irFiles: Seq[VirtualScalaJSIRFile],
+                           moduleInitializers: Seq[ModuleInitializer],
+                           logger: Logger
+                         ): List[String] = {
+    require(linkerConfig.moduleKind == ModuleKind.CommonJSModule,
+      s"linkerConfig.moduleKind was ${linkerConfig.moduleKind}")
+    val symbolRequirements = {
+      val backend = new BasicLinkerBackend(linkerConfig.semantics,
+        linkerConfig.outputMode,
+        linkerConfig.moduleKind,
+        linkerConfig.sourceMap,
+        LinkerBackend.Config())
+      backend.symbolRequirements
+    }
+    val linkingUnit =
+      linker.linkUnit(irFiles,
```

Comment:
That's not directly possible. There might be a hacky way to achieve it if you use a custom `LinkerBackend` that wraps the default one, so it can snatched the `linkingUnit` that it receives as input and stores it in some mutable state. Not pretty.